build/common.sh

```diff
@@ -1479,6 +1479,14 @@ function kube::release::docker::release() {
     docker_push_cmd=("gcloud" "docker")
   fi
 
+  if [[ "${KUBE_DOCKER_REGISTRY}" == "gcr.io/google_containers" ]]; then
+    if ! kube::release::has_gcloud_account k8s.production.user@gmail.com; then
+      kube::log::error "Pushing images to gcr.io/google_containers requires credentials for account k8s.production.user@gmail.com"
+      return 1
```

Comment:
We are already in a situation where we can just run this code twice and it will work. I moved this code to `push-official-release.sh`.